doc/source/train/examples/lightning/lightning_exp_tracking.ipynb

```diff
@@ -0,0 +1,498 @@
+    "\n",
+    "    def setup(self, trainer, pl_module, stage) -> None:\n",
+    "        wandb.login(key=self.key)\n",
+    "\n",
+    "\n",
+    "def create_loggers(name, project_name, save_dir=\"./logs\", offline=False):\n",
+    "    # Avoid creating a new experiment run on the driver node.\n",
+    "    rank_zero_only.rank = None\n",
+    "\n",
+    "    # Wandb\n",
+    "    wandb_api_key = os.environ.get(\"WANDB_API_KEY\", None)\n",
+    "    wandb_logger = WandbLogger(\n",
+    "        name=name, \n",
+    "        project=project_name, \n",
+    "        save_dir=f\"{save_dir}/wandb\", \n",
+    "        offline=offline\n",
+    "    )\n",
+    "    callbacks = [] if offline else [WandbLoginCallback(key=wandb_api_key)]\n",
+    "\n",
+    "    # CometML\n",
+    "    comet_api_key = os.environ.get(\"COMET_API_KEY\", None)\n",
+    "    comet_logger = CometLogger(\n",
+    "        api_key=comet_api_key,\n",
+    "        experiment_name=name,\n",
+    "        project_name=project_name,\n",
+    "        save_dir=f\"{save_dir}/comet\",\n",
+    "        offline=offline,\n",
+    "    )\n",
+    "\n",
+    "    # MLFlow\n",
+    "    mlflow_logger = MLFlowLogger(\n",
+    "        run_name=name,\n",
+    "        experiment_name=project_name,\n",
+    "        tracking_uri=f\"file:{save_dir}/mlflow\",\n",
+    "    )\n",
+    "\n",
+    "    # Tensorboard\n",
+    "    tensorboard_logger = TensorBoardLogger(\n",
+    "        name=name, save_dir=f\"{save_dir}/tensorboard\"\n",
+    "    )\n",
+    "\n",
+    "    return [wandb_logger, comet_logger, mlflow_logger, tensorboard_logger], callbacks"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "YOUR_SAVE_DIR = \"./logs\"\n",
+    "loggers, callbacks = create_loggers(\n",
+    "    name=\"demo-run\", project_name=\"demo-project\", save_dir=YOUR_SAVE_DIR, offline=False\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "metadata": {
+    "tags": [
+     "remove-cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# FOR SMOKE TESTS\n",
+    "loggers, callbacks = create_loggers(\n",
+    "    name=\"demo-run\", project_name=\"demo-project\", offline=True\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Train the model and check out your logs:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div class=\"tuneStatus\">\n",
+       "  <div style=\"display: flex;flex-direction: row\">\n",
+       "    <div style=\"display: flex;flex-direction: column;\">\n",
+       "      <h3>Tune Status</h3>\n",
+       "      <table>\n",
+       "<tbody>\n",
+       "<tr><td>Current time:</td><td>2023-04-28 12:46:53</td></tr>\n",
+       "<tr><td>Running for: </td><td>00:01:20.41        </td></tr>\n",
```

Comment:
We can clear the output from this last cell -- we already see the outputs from the images.

Reply:
Done!